src/core/renderers/webgl/filters/SpriteMaskFilter.js

```diff
@@ -13,6 +12,7 @@ var AbstractFilter = require('./AbstractFilter'),
 function SpriteMaskFilter(sprite)
 {
     var maskMatrix = new math.Matrix();
+    var fs = require('fs');
```

Comment:
Why was this moved out of the top requires? Should be outside the scope of this fn.